pylint/checkers/utils.py

```diff
@@ -876,7 +876,11 @@ def uninferable_final_decorators(
             except AttributeError:
                 continue
         elif isinstance(decorator, nodes.Name):
-            import_node = decorator.lookup(decorator.name)[1][0]
+            lookup_values = decorator.lookup(decorator.name)
+            if lookup_values[1]:
+                import_node = decorator.lookup(decorator.name)[1][0]
+            else:
+                continue  # pragma: no cover # Covered on Python < 3.8
```

Comment:
```suggestion
            lookup_values = decorator.lookup(decorator.name)
            if lookup_values[1]:
                import_node = lookup_values[1][0]
            else:
                continue  # pragma: no cover # Covered on Python < 3.8
```

Reply:
Thanks! Stupid mistake.. Glad I tagged you 😄